### ISSUE TITLE
Issues with standardize_parameters and haven_labelled numbers

### ISSUE THREAD

**Issue by thisisnickb** [easystats/effectsize#85]
Might be a little bit niche, but the standardize_parameters() function returns the following error message when the DV in question is a haven_labelled number (at least when used in a lmerTest model):

Error in if (.no_response_standardize(m_info) || !include_response) { : 
  missing value where TRUE/FALSE needed
In addition: Warning messages:
1: Could not access model information. 
2: Could not get model data. 

Works fine, though, if the DV is changed to a regular numeric...

**Comment by strengejacke** [comment]
Do you have a small reproducible example?

**Comment by sbw78** [comment]
I ran in to the same issue, here's a reprex:

``` r
# Generate data for reprex
library(tidyverse)
library(lme4)
#> Loading required package: Matrix
#> 
#> Attaching package: 'Matrix'
#> The following objects are masked from 'package:tidyr':
#> 
#>     expand, pack, unpack
library(report)
data <- as_tibble(ggstatsplot::Titanic_full) %>% 
  mutate(Class = labelled::to_labelled(Class),
         Survived = as.integer(Survived) - 1) 

# Model with labelled variable - throws error
model <- lme4::lmer(Survived ~ Class + Age + (1|Sex), data = data)
report(model)
#> Warning: Could not access model information.
#> Warning: Could not get model data.
#> Error in if (.no_response_standardize(m_info) || !include_response) {: missing value where TRUE/FALSE needed

# Convert labelled to numeric, everything works
data <- mutate(data, Class = as.numeric(Class))
model <- lme4::lmer(Survived ~ Class + Age + (1|Sex), data = data)
report(model)
#> We fitted a linear mixed model (estimated using REML and nloptwrap optimizer) to predict Survived with Class and Age (formula = Survived ~ Class + Age). The model included Sex as random effects (formula = ~1 | Sex). Standardized parameters were obtained by fitting the model on a standardized version of the dataset. Effect sizes were labelled following Funder's (2019) recommendations.The model's total explanatory power is substantial (conditional R2 = 0.39) and the part related to the fixed effects alone (marginal R2) is of 0.01. The model's intercept, corresponding to Survived = 0, Class = 0, Age = Adult and Sex = Female, is at 0.60 (SE = 0.23, 95% CI [0.14, 1.06], p < .05). Within this model:
#> 
#>   - The effect of Class is negative and can be considered as very small and significant (beta = -0.05, SE = 8.99e-03, std. beta = -0.11, p < .001).
#>   - The effect of AgeChild is positive and can be considered as small and significant (beta = 0.10, SE = 0.04, std. beta = 0.21, p < .05).
```

<sup>Created on 2020-06-10 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>
